rest-api/vip-endpoints.php

```diff
@@ -178,6 +179,7 @@ public function list_sites() {
 		} else {
 			// Provided for consistency, even though this provides no insightful response
 			$sites[] = array(
+				'ID' => null,
```

Comment:
Yeah, that's a good idea. I was initially thinking that since the `wp_blogs` table doesn't exist, single sites don't technically have a blog_id, but I see that `get_current_blog_id()` returns `1` on my test site.

I wonder if it's worth setting this to `get_current_blog_id()` or if we can just assume that it would always return 1 on single sites.